tests/rptest/tests/metrics_reporter_test.py

```diff
@@ -148,7 +148,9 @@ def assert_fields_are_the_same(metadata, field):
         assert last['original_logical_version'] == features[
             'original_cluster_version']
         assert last['has_valid_license']
-        assert last['has_enterprise_features'] == False
+        # NOTE: value will vary depending on FIPS mode. we're confident that
+        # the source of the value is sound, so assert on presence instead.
+        assert 'has_enterprise_features' in last
```

Comment:
is the value stable in each fips/non-fips mode? because it looks like you can test for the mode using `in_fips_environment()` from `rptest/utils/mode_checks.py`

Reply:
aha, missed that grepping around.

>is the value stable

stable relative to the node-config, yeah. less sure about the env. i seem to recall that we sometimes run tests on fips-enabled hardware w/o flipping the fips node config in redpanda... @michael-redpanda?

Reply:
cool. im fine however it turns out

Reply:
> less sure about the env. i seem to recall that we sometimes run tests on fips-enabled hardware w/o flipping the fips node config in redpanda... @michael-redpanda?

environment is stable... the `in_fips_environment()` checks for the value held in `/proc/sys/crypto/fips_enabled` to make that determination.

Reply:
Right, but in a given ducktape test, is it always the case that `fips_env -> fips_mode`?

Reply:
> Right, but in a given ducktape test, is it always the case that fips_env -> fips_mode?

Oh sorry I think I misunderstood you here.  Yes, if we are running in a FIPS environment, Redpanda will be configured to start in FIPS mode: https://github.com/redpanda-data/redpanda/blob/b91e7368c7fa6609fe3e70a67093d1f6f33b81d8/tests/rptest/services/redpanda.py#L4188-L4202